distributed/src/test/scala/asobu/distributed/EndPointSpec.scala

```diff
@@ -0,0 +1,29 @@
+package asobu.distributed
+
+import org.specs2.mutable.Specification
+import play.api.http.HttpVerbs
+
+import play.routes.compiler._
+
+object EndPointSpec extends Specification {
+  import play.api.http.HttpVerbs._
+  "Endpoint" >> {
+    val routeString =
+      """
+        |# Some Comments
+        |# Some other Comments
+        |GET   /api/dsl/factorialOf/:n   a.b.c(n: Int)
+        |
+      """.stripMargin
+
+    val parserResult = EndPoint.parse(EndPoint(routeString))
+    parserResult must beRight[Option[Route]]
+    val route: Route = parserResult.right.get.get
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of get on optional type.](https://www.codacy.com/app/kailuo-wang/asobu/file/1695331566/issues/source?bid=3037259&fileBranchId=3044589#l21)